Essentials/src/com/earth2me/essentials/utils/MaterialUtil.java

```diff
@@ -27,6 +27,8 @@
     private static final Set<Material> POTIONS;
     private static final Set<Material> SIGN_POSTS;
     private static final Set<Material> WALL_SIGNS;
+    private static final Set<Material> v1_14_DYES;
```

Comment:
```suggestion
    private static final Set<Material> V1_14_DYES;
```

\+ change any usages of this field